.gitignore

```diff
@@ -0,0 +1,5 @@
+db.sqlite3
+
+tests/__pycache__/
+
+vevn/
```

Comment:
![image](https://github.com/mate-academy/py-taxi-service-startproject/assets/93856423/f4d25eaf-0256-4fd6-81bc-86899dd264bf)

Reply:
you must add `.idea` to `.ditignore`